### DISCUSSION TITLE
Icons Component Library

### DISCUSSION THREAD

**Discussion by 1216892614** [catppuccin/catppuccin#2375] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

An icon library like lucide or heroicons. Support React usable icons and more.

I can make this lib by myself, but IDN could there any legal issues or against anyone's wishes?

It would be cool to use these icons when sharing code blocks.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Lichthagel** [comment]
https://lucide.dev/guide/basics/color (heroicons and other also use `currentColor` and/or provide color properties)

Just use https://github.com/catppuccin/palette with proper styles.

I'm unsure what exactly you do expect.
